src/framework/components/element/component.js

```diff
@@ -1444,6 +1444,10 @@ Object.defineProperty(ElementComponent.prototype, "useInput", {
             } else {
                 this.system.app.elementInput.removeElement(this);
             }
+        } else {
+            if (this._useInput === true){
+                console.warn("Elements will not get any input events because this.system.app.elementInput is not initialized")
```

Comment:
Should it be 'not created/null rather than 'not initialized'. The latter sounds like a function needs to be called on `this.system.app.elementInput` to initialize it.

Reply:
I will change it